test/commands/bot-interfaces/bot-interfaces-test-data.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+export default {
+  botInterfacesGetResponse: {
+    interface: {
+      id: "52",
```

Comment:
Use single quotes in .ts and .test.ts files. Only use double-quotes in actual .json files. Update all files here to use single quotes.